templates/react/vite.config.ts

```diff
@@ -1,7 +1,15 @@
 import { defineConfig } from "vite";
 import react from "@vitejs/plugin-react-swc";
+import path from "path";
 
-// https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
+  server: {
+    port: 3000,
+  },
+  resolve: {
+    alias: {
+      "@": path.resolve(__dirname, "src"),
```

Comment:
resolve this alias according to 

"""
replace base URL with : 

"baseUrl": ".",
    "paths": {
      "src/*": ["src/*"],
    },
    
    """ 
    
    and test the config